### PR TITLE
Zeth client flags (depends on #248)

### PR DESCRIPTION
**Pull request by dtebbs**
Improvements to  to the zeth client flags and filenames.
`send-eth` convenience command.

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -219,7 +219,7 @@ creation of Zeth notes.
 # Deposit 10 ether from `eth-address`, creating Zeth notes owned by Alice
 (env)$ zeth mix --out <public-zeth-address>,10 --vin 10
 ```
-where `<public-zeth-address>` is the contents of `zeth-address.json.pub`.
+where `<public-zeth-address>` is the contents of `zeth-address.pub`.
```
Comment (AntoineRondelet) [inline client/README.md:222]: content?

File: client/test_commands/send_eth.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    "--eth-addr",
+    help=f"Source address or filename (default: {ETH_ADDRESS_DEFAULT})")
+@option(
+    "--eth-private-key",
+    help=f"Source private key file (default: {ETH_PRIVATE_KEY_FILE_DEFAULT})")
+@option(
+    "--amount",
+    type=str,
+    default=FUND_AMOUNT_DEFAULT,
+    help=f"Amount to fund in Ethere (default: {FUND_AMOUNT_DEFAULT})")
```
Comment (AntoineRondelet) [inline client/test_commands/send_eth.py:28]: Ethers

File: client/test_commands/send_eth.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+@argument("dest-addr")
+@pass_context
+def send_eth(
+        ctx: Any,
+        dest_addr: str,
+        eth_private_key: Optional[str],
+        eth_addr: Optional[str],
+        amount: str) -> None:
+    """
+    Send ether from the local eth-addr to a destination address.
```
Comment (AntoineRondelet) [inline client/test_commands/send_eth.py:38]: Ether

File: client/test_commands/send_eth.py (hunk 2 above)
Comment (dtebbs) [reply client/test_commands/send_eth.py]: Fixed. Thanks.

File: client/test_commands/send_eth.py (hunk 3 above)
Comment (dtebbs) [reply client/test_commands/send_eth.py]: capitalized

File: client/README.md (hunk 1 above)
Comment (dtebbs) [reply client/README.md]: I think either is correct in this situation.

Comment (dtebbs) [reply client/README.md]: Actually, since we can just put the filename on the command line now I've updated the command and removed this line.

**Comment by AntoineRondelet** [comment]
LGTM

**Comment by dtebbs** [comment]
Rebased onto origin/develop

**Pull request merged**